downloader/download.go

```diff
@@ -96,7 +97,35 @@ func Download(ctx context.Context, job *library.Job) error {
 	return nil
 }
 
+func libHas(
+	ctx context.Context,
+	lib borges.Library,
+	id borges.RepositoryID,
+) (bool, borges.LocationID, error) {
+	var (
+		ok    bool
+		locID borges.LocationID
+		err   error
+		done  = make(chan struct{})
+	)
+
+	go func() {
+		ok, _, locID, err = lib.Has(id)
```

Comment:
We may think on adding context to `go-borges` as we may end up with multiple hanging goroutines. What happens if `lib.Has` returns after timeout? Writing to done is going to be blocked because there's nothing reading it?

Reply:
yes, it will. Let's make the channel buffered with capacity 1

Reply:
even better, just close the channel